packages/orbit-components/src/HorizontalScroll/README.md

```diff
@@ -19,12 +19,24 @@ After adding import into your project you can use it simply like:
 
 ## Props
 
-| Name      | Type                  | Required           | Default | Description                           |
-| --------- | --------------------- | ------------------ | ------- | ------------------------------------- |
-| minHeight | `number`              |                    |         | set minimal height                    |
-| dataTest  | `string`              |                    |         | prop for testing purposes             |
-| spacing   | [`Spacing`](#Spacing) |                    | "small" | the spacing between children elements |
-| children  | `React.ReactNode`     | :heavy_check_mark: |         | content of HorizontalScroll           |
+| Name       | Type                        | Required           | Default       | Description                               |
+| ---------- | --------------------------- | ------------------ | ------------- | ----------------------------------------- |
+| minHeight  | `number`                    |                    |               | set minimal height                        |
+| dataTest   | `string`                    |                    |               | prop for testing purposes                 |
+| spacing    | [`Spacing`](#Spacing)       |                    | "small"       | the spacing between children elements     |
+| children   | `React.ReactNode`           | :heavy_check_mark: |               | content of HorizontalScroll               |
+| useSnap    | `boolean`                   |                    |               | enable `scroll-snap-type` property        |
+| scrollSnap | [`ScrollSnap`](#ScrollSnap) |                    | "x mandatory" | set value for `scroll-snap-type` property |
```

Comment:
I think it's unnecessary to have two props for scroll snap, I'm wondering what could be a reasonable alternative 🤔  How about allowing `scrollSnap` to also be a boolean? So that if it's `true`, then it's `x mandatory`, and if it's `false` (default), then it's `none`. And besides boolean it can also take all those other values that are specified below.

What do you think?